README.md

```diff
@@ -126,3 +126,63 @@ in performance.
 
 Consult the [Getting Started](docs/GettingStarted.md) page for detailed 
 information on configuring and compiling CIRCT.
+
+## Using the Python Bindings
```

Comment:
Does this need to be in the front page of the README? IIRC we removed most of the setup steps to the GettingStarted.md guide for a reason. Personally, I don't care either way, but the cmake variables and environment variables feel a little more "getting started" content than what we currently put in the readme.